EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001358	If it was just a matter of taxing additional offerings, {nl}why doesn't the Lord Lieutenant take some measures.{nl}Anyway, how did the investigation go?{nl}
+QUEST_LV_0300_20160310_001359	So as the number of monsters increased, the strange smell started to spread.{nl}Hmm...{nl}
+QUEST_LV_0300_20160310_001360	I can't think that the monsters are not related to the smell.{nl}Can you check where that smell is coming from?{nl}
+QUEST_LV_0300_20160310_001361	If we are lucky enough, we might be able to find the source of the smell.{nl}If you find it, please bring it to me. I will get rid of it right away.
+QUEST_LV_0300_20160310_001362	I am counting on you.{nl}Please check the spots where the smell is coming from along the coast.
+QUEST_LV_0300_20160310_001363	The smell and the monsters...{nl}
+QUEST_LV_0300_20160310_001364	Just as I thought! The content of the pouch was the powder that allures monsters in.{nl}Somebody must have deliberately used them to draw the monsters in.{nl}
+QUEST_LV_0300_20160310_001365	If the smell disappears, there will be no reason for the monsters to go crazy.{nl}Can you dig out the same pockets from the pit?
+QUEST_LV_0300_20160310_001366	I will find out more about the pocket in the meantime.{nl}I don't know yet whether it suddenly came or if someone buried it.
+QUEST_LV_0300_20160310_001367	Oh my god... This many...{nl}I can see why this many monsters are coming.{nl}
+QUEST_LV_0300_20160310_001368	Someone is definitely disturbing the coral harvesting.{nl}I don't know why, but he is so evil.
+QUEST_LV_0300_20160310_001369	Yes.{nl}I told you that the Lord Lieutenant is little strange.{nl}
+QUEST_LV_0300_20160310_001370	The Lord Lieutenant going out to defeat the monsters would be quite efficient.{nl}Even if the offerings delivery are delayed a bit.{nl}
+QUEST_LV_0300_20160310_001371	Yet he didn't do it.{nl}
+QUEST_LV_0300_20160310_001372	If he knows defeating the monsters is a useless act...{nl}I think all the stories are connecting the dots.{nl}
+QUEST_LV_0300_20160310_001373	If what I am thinking is correct, the Lord Lieutenant will try to steal the tributes.{nl}However we don't have the evidence to support that claim.{nl}
+QUEST_LV_0300_20160310_001374	Alright... Let's finish our guess here. Do we lure the suspect out?{nl}I want you to be responsible for the most important role.
+QUEST_LV_0300_20160310_001375	I've told the Lord Lieutenant in advance.{nl}The smell was spreading badly in the pit, making it unable to bear.{nl}I've gotten rid of the source and they thanked me for that.{nl}
+QUEST_LV_0300_20160310_001376	In the meantime, please hide with the mayor.{nl}Of course, the soldier who will catch the culprit should go together. {nl}He should act like an observer.{nl}
+QUEST_LV_0300_20160310_001377	Of course, we can't say the soldiers have participated... But...{nl}You don't have to worry since Basilius is reliable and a very close friend of mine.{nl}
```

Comment:
"Of course, we can't rule out that some of the soldiers may have had a hand in this..."